### ISSUE TITLE
conflict with supports-screens in AndroidManifest

### ISSUE THREAD

**Issue by derekstavis** [journeyapps/zxing-android-embedded#22]
ZXing own AndroidManifest declares `supports-screens` tag that will conflict if you define the same tag in your application AndroidManifest, outputting the following errors: 

```
Error:(7, 9) Attribute supports-screens@normalScreens value=(false) from AndroidManifest.xml:7:9
Error:(9, 9) Attribute supports-screens@smallScreens value=(false) from AndroidManifest.xml:9:9
Error:Execution failed for task ':app:processDebugManifest'.
Manifest merger failed with multiple errors, see logs
```

**Comment by rkistner** [comment]
You can probably solve it by having:

     <supports-screens tools:node=”replace” ... />

That said, it will probably be best to just remove that statement from this library.

**Comment by derekstavis** [comment]
Got it working by specifying `tools:replace="android:smallScreens,android:normalScreens"`. Also needed to specify tools XML namespace to manifest tag using `xmlns:tools="http://schemas.android.com/tools"`.

**Comment by derekstavis** [comment]
I gave a look at the code and saw a ruby script called `update`. Should it be the responsible for removing `<support-screens>` statements? I'm planning a pull request on this.

**Comment by rkistner** [comment]
Fixed in #30.
